datashuttle/tui/tooltips.py

```diff
@@ -121,6 +121,11 @@ def get_tooltip(id: str) -> str:
     elif id == "#create_folders_settings_toplevel_select":
         tooltip = "The top-level-folder to create folders in."
 
+    elif id == "#suggest_next_sub_ses_remote":
+        tooltip = (
+            "Search the remote project folder for folder suggestions. "
```

Comment:
Just to add a little more detail, we could say `Search the central project folder when suggesting the next subject or session. May be slower compared than searching local only.`